requirements/requirements-app.txt

```diff
@@ -33,4 +32,39 @@ python-json-logger==2.0.7
 requests==2.31.*
 retrying==1.3.4
 urllib3==1.26.*
+
+# Opentelemetry Dependencies
+opentelemetry-api==1.28.2
+opentelemetry-distro==0.49b2
+opentelemetry-exporter-otlp==1.28.2
+opentelemetry-exporter-otlp-proto-common==1.28.2
+opentelemetry-exporter-otlp-proto-grpc==1.28.2
+opentelemetry-exporter-otlp-proto-http==1.28.2
+opentelemetry-instrumentation==0.49b2
+opentelemetry-instrumentation-asgi==0.49b2
+opentelemetry-instrumentation-asyncio==0.49b2
+opentelemetry-instrumentation-asyncpg==0.49b2
+opentelemetry-instrumentation-boto3sqs==0.49b2
+opentelemetry-instrumentation-botocore==0.49b2
+opentelemetry-instrumentation-dbapi==0.49b2
+opentelemetry-instrumentation-django==0.49b2
+opentelemetry-instrumentation-elasticsearch==0.49b2
+opentelemetry-instrumentation-grpc==0.49b2
+opentelemetry-instrumentation-logging==0.49b2
+opentelemetry-instrumentation-psycopg2==0.49b2
+opentelemetry-instrumentation-redis==0.49b2
+opentelemetry-instrumentation-requests==0.49b2
+opentelemetry-instrumentation-sqlite3==0.49b2
```

Comment:
Do we need sqlite here?

Reply:
What do you think about removing these as well?
- opentelemetry-instrumentation-asgi
- opentelemetry-instrumentation-dbapi
- opentelemetry-instrumentation-tortoiseorm